vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+---
+title: GitHub Integration
+description: Information on the available vcpkg integration points with GitHub
+author: michaelbprice
+ms.author: miprice
+ms.date: 06/15/2023
+---
+
+# Available GitHub Integrations
+
+## Caching vcpkg-built binaries for your GitHub Actions workflows
+
+Repositories that use GitHub Actions for continuous integration can take advantage of [vcpkg's binary caching feature](./users/binarycaching.md) to reduce the amount of time it takes to build their projects. There are two binary cache providers that are available to GitHub repositories.
+
+### The GitHub Actions cache provider
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+The [GitHub Actions cache](https://docs.github.com/actions/using-workflows/caching-dependencies-to-speed-up-workflows) is intended to store a repository's intermediate build files that don't change often between jobs or workflow runs. GitHub provides a few different [tools to manage your Actions caches](https://docs.github.com/actions/using-workflows/caching-dependencies-to-speed-up-workflows#managing-caches), which include REST APIs and an extension to the gh command line tool, so that you can optimize caches for your workflow. vcpkg's integration with GitHub Actions cache is through the [`x-gha` binary source provider](./users/binarycaching.md#gha).
```

Comment:
gh should be code escaped, i.e. ``gh`` The reason is so that machine translation doesn't mess with it.

Reply:
Another good catch. Fix incoming.